src/uu/chmod/src/chmod.rs

```diff
@@ -262,8 +262,10 @@ impl Chmoder {
                         );
                     }
                     return Ok(());
+                } else if err.kind() == std::io::ErrorKind::PermissionDenied {
+                    show_error!("'{}': Permission denied", file.display());
                 } else {
-                    show_error!("{}: '{}'", err, file.display());
+                    show_error!("'{}': {}", err, file.display());
```

Comment:
Doesn't the `file.display()` need to be the second argument here?

Reply:
@tertsdiepraam Thanks for the catch. Fixed in new commit.